méli-mélo/2023-09-distance-calculator/index.html

```diff
@@ -0,0 +1,309 @@
+</label>
+</li>
+<li class="radio">
+<label>
+<input type="radio" name="language" class="wb-tagfilter-ctrl" value="language__french"> French
+</label>
+</li>
+<li class="radio">
+<label>
+<input type="radio" name="language" class="wb-tagfilter-ctrl" value="language__english"> English
+</label>
+</li>
+</ul>
+</fieldset>
+</div>
+<div class="form-group mrgn-bttm-0">
+<fieldset>
+<legend class="h5 mrgn-tp-0"><span class="field-name">Event location</span></legend>
+<ul class="list-unstyled">
+<li class="checkbox">
+<label>
+<input type="checkbox" name="location" class="wb-tagfilter-ctrl" value="location__montreal"> Montreal
+</label>
+</li>
+<li class="checkbox">
+<label>
+<input type="checkbox" name="location" class="wb-tagfilter-ctrl" value="location__online"> Online
+</label>
+</li>
+<li class="checkbox">
+<label>
+<input type="checkbox" name="location" class="wb-tagfilter-ctrl" value="location__ottawa"> Ottawa
+</label>
+</li>
+<li class="checkbox">
+<label>
+<input type="checkbox" name="location" class="wb-tagfilter-ctrl" value="location__toronto"> Toronto
+</label>
+</li>
+<li class="checkbox">
+<label>
+<input type="checkbox" name="location" class="wb-tagfilter-ctrl" value="location__vancouver"> Vancouver
+</label>
+</li>
+</ul>
+</fieldset>
+</div>
+<div class="form-group mrgn-bttm-0">
+<label for="dayList">Day of the week</label>
+<select id="dayList" name="dayOTW" class="form-control full-width wb-tagfilter-ctrl">
+<option value="">All</option>
+<option value="dayOTW_monday">Monday</option>
+<option value="dayOTW_tuesday">Tuesday</option>
+<option value="dayOTW_wednesday">Wednedsday</option>
+<option value="dayOTW_thursday">Thursday</option>
+<option value="dayOTW_friday">Friday</option>
+<option value="dayOTW_saturday">Saturday</option>
+<option value="dayOTW_sunday">Sunday</option>
+</select>
+</div>
+</details>
+</div>
+<div class="col-xs-12 col-sm-8 col-lg-9">
+<ul class="list-unstyled events-list wb-tagfilter-items">
+
+<li data-wb-tags="language__english language__french location__online dayOTW_friday">
+<h3>Ask PAC Anything!</h3>
+<p><span class="event-date">Friday August 5th, 2022</span> | <span class="event-time">12:00 PM to 1:00 PM</span></p>
+<p><strong>Location:</strong> <span class="event-location">Online</span>, <strong>Language</strong>: <span class="event-language">English, French</span></p>
+</li>
+
+<li data-wb-tags="language__english location__vancouver dayOTW_wednesday">
+<h3>Bidding on Opportunities</h3>
+<p><span class="event-date">Wednesday August 10th, 2022</span> | <span class="event-time">2:00 PM to 3:30 PM</span></p>
+<p><strong>Location:</strong> <span class="event-location">Vancouver</span>, <strong>Language</strong>: <span class="event-language">English</span></p>
+<div class="distance-visible hidden"><p><strong><span class="distance" data-gc-distance-coordinates='{ "longtitude":"-123.113889","latitude":"49.261111" }'></span> Km</strong> from <span class="distance-title"></span></p></div>
```

Comment:
I did some local test and it was weird to see 3 times the city name.

![image](https://github.com/wet-boew/GCWeb/assets/1676030/c9d49c78-81ae-49fa-8d93-41a703d79200)

---

Note: Is the text in parenthesis represent the subject type, because Québec city is not the same as Québec the province. Having it rendered like in my screen capture would create confusion.

For a comparaison, here a search but where a province were detected

![image](https://github.com/wet-boew/GCWeb/assets/1676030/d369b80a-b3ee-4920-9da0-e3864933bd23)

Reply:
Parsing the returned title is something I have been considering. 

It can be more meaningful with something like Lac Gatineau returning "Lac-Gatineau, Mont-Laurier; Antoine-Labelle, Quebec (Unincorporated area)".

The text in the parenthesis appears to be the type of returned location as a whole (Town), (City), (Province), and others. So not Québec city but the returned Gatineau, Quebec is a (City).